### PR TITLE
Adds postdeploy tools

### PR DESCRIPTION
**Pull request by jcsackett**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,8 +46,9 @@ $(IMAGE): $(LXC) profile
 # Set up juju
 	$(LXC) file push ./files/juju $(INSTANCE)/usr/bin/
 
-# Set up kubectl
+# Set up kubectl and postdeploy coonfig
```
Comment (frankban) [inline Makefile]: typo: config

File: files/lshell.conf [hunk 2]
```diff
@@ -22,7 +22,7 @@ forbidden       : [';','&','`','>','<','$(','${']
 warning_counter : -1
 
 ##  command aliases list (similar to bash’s alias directive)
-aliases         : {'ll':'ls -l', 'vim':'rvim'}
+aliases         : {'ll':'ls -l', 'vim':'rvim', 'kubectl':'/snap/bin/kubectl', 'postdeploy':'/home/ubuntu/postdeploy'}
```
Comment (frankban) [inline files/lshell.conf]: Can we put postdeploy in /home/ubuntu/bin and add that to the path if not already there?
If it's trouble, nevermind.

File: files/lshell.conf [hunk 3]
```diff
@@ -8,7 +8,7 @@ loglevel        : 2
 
 [default]
 ##  a list of the allowed commands or 'all' to allow all commands in user's PATH
-allowed         : ['cat','cd','grep','echo','juju','less','ll','ls','tmux']
+allowed         : ['cat','cd','grep','echo','juju','less','ll','ls','tmux', 'kubectl', 'jq', 'postdeploy']
```
Comment (frankban) [inline files/lshell.conf]: Let's keep those in alphabetical order.

**Review by frankban — APPROVED** [review]

File: files/lshell.conf (hunk 3 above)
Comment (jcsackett) [reply files/lshell.conf]: Ok.

File: files/lshell.conf (hunk 2 above)
Comment (jcsackett) [reply files/lshell.conf]: Done.

File: Makefile (hunk 1 above)
Comment (jcsackett) [reply Makefile]: Fixed.

**Pull request closed without merging**